scripts/release

```diff
@@ -1,20 +1,28 @@
 #!/usr/bin/env bash
 set -e
 
-source $(dirname $0)/lib/version
+DEPLOY="${DEPLOY:-false}"
+
+
 source $(dirname $0)/lib/debug_functions
+source $(dirname $0)/lib/version
 
-if [[ ${VERSION} != "dev" ]]; then
-    DOCKER_TAG=${VERSION}
-else
-    DOCKER_TAG="latest"
+
+# This flag is passed from travis, so only the right jobs will deploy the
+# container images to quay
+
+if [ "$DEPLOY" != "true" ] ; then
+    echo "We don't need to deploy from this job, if you're trying to deploy manually set DEPLOY=true"
+    exit 0
```

Comment:
```suggestion

# This flag is passed from travis, so only the right jobs will deploy the
# container images to quay

if [ "$DEPLOY" != "true" ] ; then
    echo "We don't need to deploy from this job, if you're trying to deploy manually set DEPLOY=true"
    exit 0
```

Also not necessary here, 

The deploy flag is something we set from the job matrix, when we only want one job to deploy. In this case all do. And just the "condition: DEPLOY=true" will have the same effect as this (which is being removed from master, we have a PR), sorry.. :)